pages.id/common/rbenv.md

```diff
@@ -0,0 +1,36 @@
+# rbenv
+
+> Alat untuk menginstal versi Ruby dan mengatur lingkungan aplikasi dengan mudah.
+> Informasi lebih lanjut: <https://github.com/rbenv/rbenv>.
+
+- Instal suatu versi Ruby:
+
+`rbenv install {{versi}}`
+
+- Tampilkan daftar versi-versi stabil terbaru dari Ruby:
+
+`rbenv install --list`
+
+- Tampilkan daftar versi-versi Ruby yang terinstal:
+
+`rbenv versions`
+
+- Gunakan spesifik versi Ruby di seluruh sistem:
+
+`rbenv global {{version}}`
+
+- Gunakan spesifik versi Ruby untuk suatu direktori aplikasi/proyek:
+
+`rbenv local {{version}}`
+
+- Tampilkan versi Ruby yang saat ini dipilih:
+
+`rbenv version`
+
+- Hapus sebuah versi Ruby:
+
+`rbenv uninstall {{version}}`
+
+- Tampilkan semua versi Ruby yang mengandung file yang dapat dieksekusi sesuai dengan yang disebut:
+
+`rbenv whence {{executable}}`
```

Comment:
could `executable` be translated?

Reply:
We don't have a word for `executable`. The translation is `file yang dapat dieksekusi` which doesn't fit in this context.